From: Charles <charles05@canonical.com>
Date: Fri, 17 Jul 2026 00:08:58 +0100
Subject: data: Add grd user to tss group

This allows access to the TPM device for credentials storage. Otherwise,
grd will silently fallback to less secure file based credentials.

Forwarded: https://gitlab.gnome.org/GNOME/gnome-remote-desktop/-/merge_requests/408
Bug: https://gitlab.gnome.org/GNOME/gnome-remote-desktop/-/work_items/339
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+bug/2161032
---
 data/gnome-remote-desktop-sysusers.conf.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/gnome-remote-desktop-sysusers.conf.in b/data/gnome-remote-desktop-sysusers.conf.in
index 70234ec..8be2fe9 100644
--- a/data/gnome-remote-desktop-sysusers.conf.in
+++ b/data/gnome-remote-desktop-sysusers.conf.in
@@ -1,2 +1,3 @@
 # sysusers.d file to ensure the existence of the gnome-remote-desktop user
 u @GRD_USERNAME@ - "GNOME Remote Desktop" @GRD_HOMEDIR@
+m @GRD_USERNAME@ tss
